packages/graph/hash_graph/bench/benches/representative_read/mod.rs

```diff
@@ -65,41 +65,74 @@ fn bench_representative_read_multiple_entities(c: &mut Criterion) {
     group.sampling_mode(SamplingMode::Flat);
 
     let graph_resolve_depths = [
+        GraphResolveDepths::default(),
         GraphResolveDepths {
-            data_type_resolve_depth: 0,
-            property_type_resolve_depth: 0,
-            entity_type_resolve_depth: 0,
-            entity_resolve_depth: 0,
-        },
-        GraphResolveDepths {
-            data_type_resolve_depth: 0,
-            property_type_resolve_depth: 0,
-            entity_type_resolve_depth: 0,
-            entity_resolve_depth: 2,
+            inheritance_resolve_depth: 0,
```

Comment:
Can we rename these to be exactly equivalent to the edge kinds, so `inherits_from` for this